todoman/cli.py

```diff
@@ -171,10 +171,12 @@ def cli(ctx, color, porcelain):
 @click.argument('summary', nargs=-1)
 @click.option('--list', '-l', callback=_validate_list_param,
               help='The list to create the task in.')
+@click.option('--category',
+              help='Categories of todo')
```

Comment:
Please change this to `A comma-separated list of categories`, since the comma-separated bit won't be obvious to users (though it definitely is to you and me 😄  ).